### PR TITLE
[ISSUES #132] enhance health check

### LINKED ISSUE #132: Enhance: 健康检查机制优化

### PR DESCRIPTION
**Pull request by onewe**
- optimize payload convert logic
- add ErrResponse error
- add health check task
- optimize bi default handle logic

Close #132

### CODE REVIEW

File: src/common/remote/grpc/mod.rs [hunk 1]
```diff
@@ -97,9 +104,78 @@ impl NacosGrpcClient {
...
+            return Ok(());
+        }
+
+        // got permit , health check.
+        let health_check_request = HealthCheckRequest::default();
+        let response = self
+            .unary_call_async::<HealthCheckRequest, HealthCheckResponse>(health_check_request)
+            .await;
+        if response.is_ok() {
+            warn!("the current client health check pass, don't need to reinitialize");
```
Comment (CherishCai) [inline src/common/remote/grpc/mod.rs:176]: 正常情况不应该 warn

File: src/common/remote/grpc/mod.rs [hunk 2]
```diff
@@ -97,9 +104,78 @@ impl NacosGrpcClient {
...
+                        _ => {
+                            //just ignore error
+                            error!("health check failed. {e}");
+                            tokio::time::sleep(Duration::from_secs(5)).await;
+                            continue;
+                        }
+                    }
+                }
+                let response = response.unwrap().into_body();
+                info!("health check. {response:?}");
```
Comment (CherishCai) [inline src/common/remote/grpc/mod.rs:155]: info health check 的话日志太频繁了，建议 debug

File: src/common/remote/grpc/handler/default_handler.rs [hunk 3]
```diff
@@ -1,12 +1,42 @@
...
+        let p_type;
+        let r_body;
+        let r_type;
+        let client_ip;
+        let headers;
+
+        if let Some(body) = payload.body {
+            p_type = body.type_url;
+            let body_str = String::from_utf8(body.value);
+            if let Err(e) = body_str {
```
[the remark this answers is not in the document]
Comment (onewe) [reply src/common/remote/grpc/handler/default_handler.rs:19]: unwrap_or 之后没办法打印错误信息

**Review by CherishCai — APPROVED** [review]
LGTM

**Pull request merged**